pkg/event/event_unmarshal_test.go

```diff
@@ -57,7 +58,7 @@ func TestUnMarshal(t *testing.T) {
 							Resource:  resource,
 							DataType:  event.NOTIFICATION,
 							ValueType: event.ENUMERATION,
-							Value:     event.FREERUN,
+							Value:     string(ptp.FREERUN),
```

Comment:
instead of casting to string add this to enum
```

func (e MyEnum) String() string {
    switch e {
    case Foo:
        return "Foo"
    case Bar:
        return "Bar"
    default:
        return fmt.Sprintf("%d", int(e))
    }
}

```

Reply:
refactored to use iota enum for SyncState

Reply:
done